skycoin/skywire
# Allow to configure the vpn apps from the manager UI

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

 Changes:	
- The modal windows used for configuring the proxy apps now work for configuring the vpn apps too.
- Now it is possible to filter by country the entries returned by the discovery service.
- Flags were added to allow to easily identify the countries. Also, the ISO codes are now converted to the full name of the countries.

How to test this PR:
Using the Skywire manager, enter to the app list of a visor and press the small config button for the vpn server and client apps.

NOTE: the same methods used for configuring the skysocks apps were used for configuring the vpn apps. Please check if the correct params are being updated in the configuration file after changing them using the manager UI.
